homeassistant/__main__.py

```diff
@@ -274,7 +274,7 @@ def try_to_restart() -> None:
     # thread left (which is us). Nothing we really do with it, but it might be
     # useful when debugging shutdown/restart issues.
     try:
-        nthreads = sum(thread.isAlive() and not thread.isDaemon()
+        nthreads = sum(thread.is_alive() and not thread.isDaemon()
```

Comment:
Weird thing is that it hasn't been deprecated per se, right now it is a "please use daemon property".